Cesium.CodeGen/Ir/Expressions/FunctionPointerValue.cs

```diff
@@ -0,0 +1,30 @@
+using Cesium.CodeGen.Contexts;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+namespace Cesium.CodeGen.Ir.Expressions;
+
+internal class FunctionPointerValue : IValue
+{
+    private readonly MethodReference _methodReference;
+
+    public FunctionPointerValue(MethodReference methodReference)
+    {
+        _methodReference = methodReference;
+    }
+
+    public void EmitGetValue(IDeclarationScope scope)
+    {
+        throw new NotImplementedException();
```

Comment:
I need some help with implementation of this method.

Reply:
Alright; I'm on it.